system-addon/lib/Screenshots.jsm

```diff
@@ -63,6 +67,24 @@ this.Screenshots = {
     return screenshot;
   },
 
+  /**
+   * Checks if all the open windows are private browsing windows. If so, we do not
+   * want to collect screenshots. If there exists at least 1 non-private window,
+   * we are ok to collect screenshots.
+   */
+  _shouldGetScreenshots() {
+    const windows = Services.wm.getEnumerator("navigator:browser");
+    let shouldGetScreenshots = false;
+    while (windows.hasMoreElements()) {
+      if (!PrivateBrowsingUtils.isWindowPrivate(windows.getNext())) {
+        // As soon as we encounter 1 non-private window, screenshots are fair game.
+        shouldGetScreenshots = true;
+        break;
```

Comment:
nit: could just conditionally `return true;` and `return false;` at the end without the variable or `break`.